demo/src/library.json

```diff
@@ -369,16 +369,16 @@
         "description": "Adds a label tag above Component if given."
       },
       {
-        "name": "list (optional format 1)",
+        "name": "children",
         "type": "Array",
-        "default": "[]",
-        "description": "Format: [ 'item 1', 'item 2' ] (if no list prop given, Component will return null)."
+        "default": "null",
+        "description": "Format: [<div />, <div />, <div />] (if no children are provided, Component will return null)."
       },
       {
-        "name": "list (optional format 2)",
-        "type": "Array",
-        "default": "[]",
-        "description": "Format: [{ text: 'Option 1, value: 1 }, { text: 'Option 2', value: 2 }] (if no list prop given, Component will return null)."
+        "name": "value (optional)",
+        "type": "string/number",
+        "default": "null",
+        "description": "Passing a value prop  will make this component a controlled component"
```

Comment:
@azamatsmith I would also throw an example in the description of what the Array should look like too. 👍